src/constants/notificationSchema.js

```diff
@@ -70,30 +72,33 @@ notificationSchemaPlaceholder[
   titleId: 'COLLABORATION_EDIT_REVOKE_TITLE',
   notificationMessage: 'EDIT_COLLABORATION_REVOKED',
   moreDetailedDescription: 'EDIT_COLLABORATION_REVOKED',
-  viewInNotificationPane: false,
+  showNotificationDialog: false,
+  buttonPath: '/',
 };
 notificationSchemaPlaceholder[
   notificationTypeNames.collaboration_manager_revoke
 ] = {
   titleId: 'COLLABORATION_MANAGER_REVOKE_TITLE',
   notificationMessage: 'COLLABORATION_REVOKED_BY_MANAGER',
   moreDetailedDescription: 'COLLABORATION_REVOKED_BY_MANAGER',
-  viewInNotificationPane: false,
+  showNotificationDialog: false,
+  buttonPath: '/',
 };
 notificationSchemaPlaceholder[
   notificationTypeNames.individual_merge_request
 ] = {
   titleId: 'INDIVIDUAL_MERGE_REQUEST_TITLE',
   notificationMessage: 'INDIVIDUAL_MERGE_REQUEST_MESSAGE',
   moreDetailedDescription: 'INDIVIDUAL_MERGE_REQUEST_MESSAGE',
-  viewInNotificationPane: true,
+  showNotificationDialog: true,
 };
 notificationSchemaPlaceholder[
   notificationTypeNames.individual_merge_complete
 ] = {
   titleId: 'INDIVIDUAL_MERGE_COMPLETE_TITLE',
   notificationMessage: 'INDIVIDUALS_MERGE_COMPLETE_MESSAGE',
   moreDetailedDescription: 'INDIVIDUALS_MERGE_COMPLETE_MESSAGE',
-  viewInNotificationPane: false,
+  showNotificationDialog: false,
+  buttonPath: '/individuals/', // TODO figure out how to pass individual ID to this
```

Comment:
These TODOs having to do with individual notifications will be fleshed out once we can use the UI to merge individuals.

Reply:
yep all the merge notifications will need to get worked on once you can actually merge. this is fine for now :)